kratos/sources/kratos_parameters.cpp

```diff
@@ -1088,6 +1088,19 @@ void Parameters::Append(const Parameters& rValue)
 /***********************************************************************************/
 /***********************************************************************************/
 
+void Parameters::CopyValuesFromExistingParameters(
+    const Parameters OriginParameters,
+    std::vector<std::string>& rListParametersToCopy
+    )
+{
+    for (auto& r_value_name : rListParametersToCopy) {
+        this->AddValue(r_value_name, OriginParameters[r_value_name]);
```

Comment:
Would it be possible to check whther the `r_value_name` is in `OriginParameters`, if not print the `OriginParameters` and stating following `rListParametersToCopy` is required. This would be very useful information for the normal user.